packages/hebao_v1/contracts/modules/security/GuardianModule.sol

```diff
@@ -48,6 +59,17 @@ contract GuardianModule is SecurityModule
         pendingPeriod = _pendingPeriod;
     }
 
+    function bindableMethods()
+        public
+        pure
+        override
+        returns (bytes4[] memory methods)
+    {
+        methods = new bytes4[](2);
+        methods[0] = this.getLock.selector;
+        methods[1] = this.isLocked.selector;
```

Comment:
Maybe these not so useful binded methods can also be removed as they are expensive to set. I don't really see a big need to be able to do `wallet.isLocked` or `wallet.getLock`, may as well just call the methods on the module/storage for that. It's a non-standard interface (unlike `isValidSignature`) that 3rd party contracts that interact directly with the wallet won't depend on.

Reply:
Ok, will do